lib/std/os/bits/linux/mips.zig

```diff
@@ -383,6 +383,9 @@ pub const SYS = extern enum(usize) {
     statx = Linux + 366,
     rseq = Linux + 367,
     io_pgetevents = Linux + 368,
+    io_uring_setup = Linux + 425,
+    io_uring_enter = Linux + 426,
+    io_uring_register = Linux + 427,
```

Comment:
Could you send this as a separate PR (which can hopefully be merged quickly); also including any other missing syscalls on mips in a similar range?

Reply:
I don't think that's necessary. This PR is ready to merge (once the build passes finishes).

Reply:
I will do a separate PR for the other missing mips syscalls after this, as soon as I have time.